Assets/Prefabs/Num0.prefab

```diff
@@ -0,0 +1,82 @@
+%YAML 1.1
```

Comment:
I strongly recommend (not for this PR but in the very near future), reducing these number prefabs to a single GameObject that displays it's number dynamically. That is, add a textComponent that displays the number the block stores.

You don't want to have to maintain a library of 1000 number sprites. If we want to ever change the design, we'd be screwed.